Formula/urlview.rb

```diff
@@ -2,9 +2,17 @@ class Urlview < Formula
   desc "URL extractor/launcher"
   homepage "https://packages.debian.org/sid/misc/urlview"
   url "https://deb.debian.org/debian/pool/main/u/urlview/urlview_0.9.orig.tar.gz"
-  version "0.9-20"
+  version "0.9-21"
```

Comment:
so the sha did not get changed btw the two versions?

Reply:
In this formula's current state, the `stable` archive stays the same and the formula is updated by applying the latest patch. If there's ever a new major/minor version, then we would change the `stable` archive and the `sha256` would change then.

If you look lower in the changes, you'll see that the patch URL changed (bumped from `0.9-20` to `0.9-21`) and that `sha256` changed.

Reply:
got you. make sense.